tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -83,7 +83,6 @@ def _process_and_assert_configuration_changes(
         frames: Sequence[Mapping[str, str | bool]],
         platform: str,
         expected_new_code_mappings: Sequence[ExpectedCodeMapping] | None = None,
-        expected_num_code_mappings: int = 1,
```

Comment:
This had been used in the tests to check the total number of code mappings expected. For dry run tests, it would always be 0 but we already check that no new code mappings are being created with `assert len(dry_run_code_mappings) == len(expected_new_code_mappings)`.